data/data/openstack/main.tf

```
@@ -40,6 +40,7 @@ module "bootstrap" {
   master_port_ids         = module.topology.master_port_ids
   root_volume_size        = var.openstack_master_root_volume_size
   root_volume_type        = var.openstack_master_root_volume_type
+  zone                    = var.openstack_master_availibility_zones[0]
```

What happens when openstack_master_availibility_zones is empty ?

Feels like it should be:
```suggestion
  zone                    = length(var.openstack_master_availibility_zones) > 0 ? var.openstack_master_availibility_zones[0] : ""
```

---

I have this logic in the master code: 

```hcl
resource "openstack_compute_instance_v2" "master_conf" {
  name = "${var.cluster_id}-master-${count.index}"
  count = var.instance_count

  flavor_id = data.openstack_compute_flavor_v2.masters_flavor.id
  image_id = var.root_volume_size == null ? var.base_image_id : null
  security_groups = var.master_sg_ids
  availability_zone = length(var.zones) > 0 ? var.zones[count.index % length(var.zones)] : ""
```

---

ah but it will null ptr